modules/prokka/prokka.nf

```diff
@@ -25,7 +25,7 @@ process PROKKA {
 
     prokka_gramstain_argument = ""
 
-    if( params.prokka_gram ) {
+    if( params.prokka_signal_peptides ) {
```

Comment:
We must not forget to also update the docs regarding the new parameter name.

Reply:
Mm, it's not mentioned in the docs at the moment. https://github.com/ctmrbio/BACTpipe/search?q=prokka_gram 

@thorellk could you please add the docs within this branch (and therefore the PR) ? 😊

Reply:
Added a reference to doc here.
https://github.com/ctmrbio/BACTpipe/pull/161/commits/8cbd0e064d5ef965b2929152eaf0ec6e5d3f1349